elm/compile.py

```diff
@@ -30,6 +30,8 @@
         os.path.join(internal_package_dir, metadata["version"]),
     )
 
+if len(sys.argv) <= 8:
+    os.makedirs(PACKAGES_DIR)
```

Comment:
The if-statement seems a bit magical here. Could you maybe remove the if-statement, and instead call os.makedirs() unconditionally right after line 7 of this script? Thanks!

Reply:
Done (sorry, just saw this). I'm unable to test it, but I can't see why it wouldn't still work, as the change is purely aesthetic.